spring-cloud-dataflow-scheduler-launcher/src/main/docker/assembly.xml

```diff
@@ -0,0 +1,38 @@
+<!--
+  ~ Copyright 2019 the original author or authors.
+  ~
+  ~  Licensed under the Apache License, Version 2.0 (the "License");
+  ~  you may not use this file except in compliance with the License.
+  ~  You may obtain a copy of the License at
+  ~
+  ~          https://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~  Unless required by applicable law or agreed to in writing, software
+  ~  distributed under the License is distributed on an "AS IS" BASIS,
+  ~  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~  See the License for the specific language governing permissions and
+  ~  limitations under the License.
+  -->
+
+<assembly
+		xmlns="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2"
+		xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		xsi:schemaLocation="http://maven.apache.org/plugins/maven-assembly-plugin/assembly/1.1.2 https://maven.apache.org/xsd/assembly-1.1.2.xsd">
+	<id>pring-cloud-dataflow-scheduler-task-launcher</id>
+	<dependencySets>
+		<dependencySet>
+			<includes>
+				<include>org.springframework.cloud:spring-cloud-dataflow-scheduler-task-launcher</include>
+			</includes>
+			<outputDirectory>maven</outputDirectory>
+			<outputFileNameMapping>scheduler-task-launcher.jar</outputFileNameMapping>
+		</dependencySet>
+	</dependencySets>
+	<files>
+		<file>
+			<source>./target/spring-cloud-dataflow-scheduler-task-launcher-${project.version}.jar</source>
+			<outputDirectory>.</outputDirectory>
+			<destName>shell.jar</destName>
```

Comment:
Not sure why is this `shell.jar`.

Reply:
I think skeleton was copied from shell module :)